src/readii/image_processing.py

```
@@ -181,15 +184,23 @@ def displayImageSlice(
         image = sitk.GetArrayFromImage(image)
+        # Once converted to an array, sliceDim is always first
+        sliceDim = "first"
 
     # Get min and max value from image to define range in display
     if dispMin == None:
         dispMin = image.min()
     if dispMax == None:
         dispMax = image.max()
 
+    # Get the slice of the image to display, depending on sliceDim
+    if sliceDim == "first": dispSlice = image[sliceIdx, :, :]
```

Ruff (0.8.0)

Multiple statements on one line (colon)

(E701)